examples/tts/conf/rad-tts_feature_pred.yaml

```diff
@@ -51,7 +52,25 @@ model:
       verbose: false
       punct_pre_process: true
       punct_post_process: true
-      
+  
+  text_tokenizer:
+        _target_: "nemo.collections.tts.torch.tts_tokenizers.EnglishPhonemesTokenizer"
+        punct: True
+        stresses: True
+        chars: True
+        space: ' '
+        silence: null
+        apostrophe: True
+        sep: '|'
+        add_blank_at: null
+        pad_with_space: True
```

Comment:
Is there a reason that the params for the two rad-tts yamls are different?

Reply:
@MikyasDesta : ^^^^^^

Reply:
No. There is no reason. Let's keep them the same. Let's choose rad-tts_dec's parameter for now. I can update my branch.